src/analytics.js

```diff
@@ -61,6 +67,7 @@ function getInterviewerPlayerEventProperties(target) {
 
 
 // Take the target of an event and return an object of the relevant properties to be included in the tracking event.
+// Return `null` if the event should not be sent.
 function getEventProperties(eventName, target) {
     const eventProperties = {}
```

Comment:
How does `block` get added to `eventProperties`? I see that `getFAQEventProperties(target)` adds it but not sure about the other blocks? Are the block names specified as `data-event-block` attributes and then handled L76?
If so, it might be worth a comment

Reply:
You got it! Will add a comment :)